documentation/modules/overview/con-configuration-points-listener-names.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is included in:
+//
+// assembly-deploy-client-access.adoc
+
+[id="con-configuration-points-listener-names-{context}"]
+= Listener naming conventions
+
+[role="_abstract"]
+From the listener configuration, the resulting listener bootstrap and broker names are structured according to the following naming conventions:
+
+.Listener naming conventions
+[cols="2a,4,4", options="header"]
+|===
+| Listener type | Bootstrap name | Broker name
+| `internal` | <cluster_name>-kafka-bootstrap | _Not applicable_
+| `external` | <cluster_name>-kafka-<listener-name>-bootstrap | <cluster_name>-kafka-<listener-name>-<idx>
```

Comment:
We should list the types as there is not a real `external` type anymore (+ `cluster-ip` which would belong here is not external really)

```suggestion
| `loadbalancer` / `nodeport` / `ingress` / `route` / `cluster-ip` | <cluster_name>-kafka-<listener-name>-bootstrap | <cluster_name>-kafka-<listener-name>-<idx>
```